### PR TITLE
update devtools remote debugging get started doc

### PR DESCRIPTION
**Pull request by kaycebasques**
Fixes some old screenshots and instructions.

Also includes a feedback approach that I'm experimenting with.

### CODE REVIEW

File: src/content/en/tools/chrome-devtools/_toc.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ toc:
         path: /web/tools/chrome-devtools/device-mode/testing-other-browsers
     - title: Remote Debugging Android Devices
       section:
-      - title: Overview
+      - title: Get Started
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/_toc.yaml]: Change this back to overview and the in-page title back to 'Remote Debugging Android Devices'. This is consistent with the rest of the site.

File: src/content/en/tools/chrome-devtools/remote-debugging/index.md [hunk 2]
```diff
@@ -2,151 +2,152 @@ project_path: /web/_project.yaml
...
-dialog. 
+1. If your device is showing up as **Unknown**, accept the **Allow USB
+   Debugging** permission prompt on your Android device. **Unknown** is
+   replaced with the model name of your Android device. The green circle
+   and the **Connected** text indicate that you are all set to remotely
+   debug your Android device from your development machine.
 
-![inspect devices dialog with connection](imgs/connected-inspect-devices.png)
+Note: If you have any issues during the discovery process, you 
+can restart the process by selecting **Settings** > **Developer Options** >
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/remote-debugging/index.md]: nit: 'you can restart it'.

File: src/content/en/tools/chrome-devtools/remote-debugging/index.md [hunk 3]
```diff
@@ -2,151 +2,152 @@ project_path: /web/_project.yaml
...
+**Revoke USB Debugging Authorizations** on your Android device.
 
-## Debug content on Android device from development machine
+## Step 2: Debug content on Android device from development machine {#debug}
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/remote-debugging/index.md]: 'from your development machine' or 'from a development machine'.

File: src/content/en/tools/chrome-devtools/remote-debugging/index.md [hunk 4]
```diff
@@ -2,151 +2,152 @@ project_path: /web/_project.yaml
...
 
-From the Inspect Devices dialog, select your device from the menu on the left.
+1. If you don't already have Chrome open on your Android device, open it now.
 
-![inspecting a device](imgs/selected-inspect-device.png)
+1. Back in DevTools, click on the tab that matches your device's
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/remote-debugging/index.md]: 'Click the tab', not 'click on the tab'. This is a Google rule. Also Strunk and White. Do throughout.

File: src/content/en/tools/chrome-devtools/remote-debugging/index.md [hunk 5]
```diff
@@ -2,151 +2,152 @@ project_path: /web/_project.yaml
...
-To reload, focus, or close an open tab, click the **more options** icon next
-to the **inspect** button. 
+1. Click **Inspect** next to the URL that you just opened. A new DevTools
+   instance opens. The version of Chrome running on your Android device
+   determines the version of DevTools that opens on your development machine.
+   So, if your Android device is running a very old version of Chrome, the
+   DevTools instance may look very different than what you're used to.
 
-![reload, focus, or close a tab](imgs/more-options.png)
+### More actions: Reload, focus, or close a tab {#more-actions}
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/remote-debugging/index.md]: 'Reload' should be lowercase.

File: src/content/en/tools/chrome-devtools/remote-debugging/index.md [hunk 6]
```diff
@@ -2,151 +2,152 @@ project_path: /web/_project.yaml
...
-after the first touch, so you need to re-enable it every time that you want 
-to use this feature. 
+Go to the **Elements** panel of your DevTools instance, and hover over an
+element to highlight it in the viewport of your Android device.
 
-## Screencast from Android device to development machine
+You can also tap on your Android device screen to select an element in the
+**Elements** panel. Click **Select Element** ![Select
+Element][select]{:.devtools-inline} on your DevTools instance, and then tap
+on the element on your Android device screen. Note that **Select Element**
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/remote-debugging/index.md]: 'Tap the element', not 'tap on the element.'

File: src/content/en/tools/chrome-devtools/remote-debugging/index.md [hunk 7]
```diff
@@ -2,151 +2,152 @@ project_path: /web/_project.yaml
...
+you want to use this feature.
 
-Enable the **toggle screencast** button 
-(![toggle screencast button](imgs/toggle-screencast.png){:.inline})
-to view a screencast of the content on your Android device from within your 
-DevTools window. 
+[select]: imgs/select-element.png
 
-![screencasted android device content in devtools](imgs/screencast.png)
+### Screencast from Android device to development machine {#screencast}
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/remote-debugging/index.md]: to a development machine

File: src/content/en/tools/chrome-devtools/remote-debugging/index.md [hunk 8]
```diff
@@ -156,3 +157,53 @@ You can interact with the screencast in multiple ways:
 * To scroll, use your trackpad or mouse wheel, or fling with your mouse
   pointer.
 
+Some notes on screencasts:
+
+* Screencasts only display page content. Transparent portions of the screencast 
+  represent device interfaces, such as the Chrome omnibox, the Android status 
+  bar, or the Android keyboard. 
+* Screencasting continuously capture frames, so disable it if your testing
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/remote-debugging/index.md]: 'Screencasting continuously capture frames'

Something's not right with this phrase. Can you please revisit.

File: src/content/en/tools/chrome-devtools/_toc.yaml (hunk 1 above)
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/_toc.yaml]: This doc is a Get Started. Check the DevRel doc types. I'll be updating all the DevTools docs to be more consistent with those types

cc @Meggin

File: src/content/en/tools/chrome-devtools/remote-debugging/index.md (hunk 8 above)
Comment (petele) [inline src/content/en/tools/chrome-devtools/remote-debugging/index.md]: It makes sense to me, but I can see Joe's point. Maybe phrase as "Screencasting continually captures frames, which may affect the performance of your app." 

In any event, your/you're ;)

**Comment by petele** [comment]
@kaycebasques - when this is ready to go, please remove the DO NOT MERGE tag and I'll merge.

**Comment by petele** [comment]
@kaycebasques - looks like there's a build error. See https://ci.cloudware.io/google/WebFundamentals/4738

File: src/content/en/tools/chrome-devtools/remote-debugging/index.md (hunk 7 above)
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/remote-debugging/index.md]: ✔️

**Comment by kaycebasques** [comment]
Thanks for writerly review, @jpmedley. Updated per feedback.

After reviews are done let me rebase the commits before we merge

**Comment by jpmedley** [comment]
@kaycebasques Please do that now.

**Comment by kaycebasques** [comment]
@jpmedley done

**Pull request merged**